docs/standalone/batches.md

```diff
@@ -1,3 +1,55 @@
 ### Batches
 
-Batches can't currently be primed. See Activity verification for details of how to verify which batches have been executed.
\ No newline at end of file
+As of version ```0.1.0``` you can prime batches based on the queries they contain, the consistency and the batch type (LOGGED, UNLOGGED, COUNTER)
```

Comment:
Could be good to mention LOGGED is default that will be matched against, although maybe that is implied since that is the default in C* as well.